js/store.js

```diff
@@ -251,19 +251,23 @@ const store = {
           break;
         case 'isVisible':
           if ('isVisible' in website
-              && website.isVisible === true) {
-            break;
+              && website.isVisible === true
+              && value === 0) {
```

Comment:
I'm lost, isn't this doing exactly the same other than allowing perhaps different values later?